modules/php/States/SelectCardTrait.php

```diff
@@ -44,4 +45,18 @@ public function select($ids)
     }
     self::reactAux(Players::get($playerId), $ids);
   }
+
+  public function stSelect() {
+    $player = Players::get(Stack::top()['pId']);
+    if ($player->getGeneralStorePref()) {
```

Comment:
Shouldn't you check that the atom correspond to a general store ? Or if it's more general (for lucky duke for instance), you could call the user pref simply "auto pick for selection modal"